app/models/alchemy/page.rb

```diff
@@ -118,6 +118,9 @@ class Page < ActiveRecord::Base
       if: -> { public_on.present? && published_at.nil? },
       unless: :systempage?
 
+    before_save :set_fixed_atrributes,
+      if: -> { fixed_attributes.any? }
```

Comment:
Depends: I want to ensure, that these values never change. Not only on create. Never. So, why add another conditional? It doesn't hurt to set them on every save. It's an attribute setter, nothing else involved. This is the most effective implementation.

Will fix the name, tough